DH_Engine/Classes/DHVotingHandler.uc

```diff
@@ -430,7 +433,7 @@ function string SetupGameMap(MapVoteMapList MapInfo, int GameIndex, MapHistoryIn
     // Add Per-GameType Game Options
     if (GameConfig[GameIndex].Options != "")
     {
-        OptionString = OptionString $ Repl(Repl(GameConfig[GameIndex].Options,",","?")," ","");
+        OptionString = OptionString $ Repl(Repl(GameConfig[GameIndex].Options, ", ", "?"), " ", "");
```

Comment:
It's not, you had `","`, then changed it to `", "` (with a space). This changes the logic.